src/CWSimulationBridge.ts

```diff
@@ -192,10 +193,10 @@ export default class CWSimulationBridge {
     return result;
   }
 
-  async query(contractAddress: string, msg: any) {
+  async query(contractAddress: string, msg: any, step:string) {
```

Comment:
please, instead of `step: string`, do `trace: TraceLog`, which is less ambiguous

Reply:
Actually I did pass step because the contract info was already being extracted in query function, and if I have to pass trace then I need to extract this info inside Query Component as well. So, in order to avoid that I passed in step and which in turns utilises the contract info inside query fun in order to get trace.

Reply:
Ohhh, I see. Gotcha, yeah that makes sense. Thanks.